cmd/config.go

```diff
@@ -140,6 +142,10 @@ func (config *Config) Print() {
 		glog.Infof("Mempool Dump Directory: %s", config.MempoolDumpDirectory)
 	}
 
+	if config.PostgresURI != "" {
```

Comment:
Please make sure n0 and n0_test are working after this change with proper default flags and whatnot.